catroid/src/main/java/org/catrobat/catroid/content/actions/HideTextAction.java

```diff
@@ -48,6 +52,22 @@ protected void begin() {
 		List<UserVariable> variableList = dataContainer.getProjectVariables();
 		Map<Sprite, List<UserVariable>> spriteVariableMap = dataContainer.getSpriteVariableMap();
 
+		if (StageActivity.stageListener != null) {
+			Array<Actor> stageActors = StageActivity.stageListener.getStage().getActors();
+			ShowTextActor dummyActor = new ShowTextActor(new UserVariable("dummyActor"), 0, 0, sprite, userBrick);
+
+			for (Actor actor : stageActors) {
+				if (actor.getClass().equals(dummyActor.getClass())) {
+					ShowTextActor showTextActor = (ShowTextActor) actor;
+					if (showTextActor.getVariableNameToCompare().equals(variableToHide.getName())
+							&& showTextActor.getSprite().equals(sprite)
+							&& (userBrick != null ? showTextActor.getUserBrick().equals(userBrick) : true)) {
+						actor.remove();
+					}
+				}
+			}
+		}
```

Comment:
c&p code from https://github.com/Catrobat/Catroid/blob/390af3f7e8398dc740fd69830415e5af3490ecf0/catroid/src/main/java/org/catrobat/catroid/content/actions/ShowTextAction.java#L70-L83

I'm fine with that!